### PR TITLE
Created a css loader and improved the signup drop down menu

### PR DESCRIPTION
**Pull request by HephzibahDesigns**

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -4,10 +4,12 @@ import styles from './Button.module.scss'
 
 type ButtonProps = {
   element?: ReactNode
+  type?: 'button' | 'submit' | 'reset'
```
Comment (mosidrum) [inline src/components/Button/Button.tsx:7]: I Like how you added this type props, but it should a type of `"Submit" | "Reset"` and the default should be "Submit"

File: src/components/InputField/TextInput.module.scss [hunk 2]
```diff
@@ -12,7 +12,7 @@
 .label {
   display: flex;
   font-weight: 500;
-  font-size: 12px;
+  font-size: 16px;
```
Comment (mosidrum) [inline src/components/InputField/TextInput.module.scss:15]: why did you change this?, Although we were suppose to have text components for each font size used in the whole project

File: src/pages/Auth/DropDown.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  label: string
+}
+
+interface CustomDropdownProps extends FieldProps {
+  label: string
+  options: Option[]
+}
+
+const DropDown: React.FC<CustomDropdownProps> = ({ label, options, field }) => {
+  const [isOpen, setIsOpen] = useState(false)
```
Comment (mosidrum) [inline src/pages/Auth/DropDown.tsx:15]: Can you rename this "SelectUserType", since its not used anywhere else, I think DropDown is generic and not specific

File: src/components/InputField/TextInput.module.scss (hunk 2 above)
Comment (HephzibahDesigns) [reply src/components/InputField/TextInput.module.scss:15]: it was quite small

File: src/pages/Auth/Loading.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+const Loading = () => {
+  return (
+    <div className="flex justify-center items-center border-[1px] rounded-lg bg-white">
+      <ColorRing
+        visible={true}
+        height="50"
+        width="50"
+        ariaLabel="color-ring-loading"
+        wrapperStyle={{}}
```
Comment (mosidrum) [inline src/pages/Auth/Loading.tsx:12]: kindly remove this, since its not used

File: src/components/InputField/TextInput.module.scss (hunk 2 above)
Comment (HephzibahDesigns) [reply src/components/InputField/TextInput.module.scss:15]: and it wasn't changing sef, i wanted the label to corrolate with the font size of the register

File: src/pages/Auth/Loading.tsx (hunk 4 above)
Comment (HephzibahDesigns) [reply src/pages/Auth/Loading.tsx:12]: it's been used 
tha's the css loader

File: src/pages/Auth/SignIn.tsx [hunk 5]
```diff
@@ -155,14 +163,19 @@ const SignIn = () => {
...
+                  <div className="flex my-2">
+                    <Field
+                      name="userType"
+                      component={DropDown}
+                      label="Register as :"
+                      options={[
+                        { value: '', label: 'Choose an option' },
+                        { value: 'talent', label: 'Talent' },
+                        { value: 'recruiter', label: 'Recruiter' },
+                      ]}
```
Comment (mosidrum) [inline src/pages/Auth/SignIn.tsx:175]: Move this objects outside, and just call it in

File: src/pages/Auth/DropDown.tsx (hunk 3 above)
Comment (HephzibahDesigns) [reply src/pages/Auth/DropDown.tsx:15]: that's the name of the function

**Review by mosidrum — CHANGES_REQUESTED** [review]
Just minor changes

File: src/pages/Auth/Loading.tsx (hunk 4 above)
Comment (mosidrum) [reply src/pages/Auth/Loading.tsx]: remove the `wrapperStyle`

File: src/pages/Auth/SignIn.tsx (hunk 5 above)
Comment (HephzibahDesigns) [reply src/pages/Auth/SignIn.tsx:175]: keeping it simple it's the best, other than mapping through it again

**Comment by HephzibahDesigns** [comment]
i have made the required changes

File: src/components/Button/Button.tsx (hunk 1 above)
Comment (HephzibahDesigns) [reply src/components/Button/Button.tsx]: done

File: src/pages/Auth/SignIn.tsx (hunk 5 above)
Comment (mosidrum) [inline src/pages/Auth/SignIn.tsx:175]: You didnt work on this

create  
```
userTypeData = [
     { value: '', label: 'Choose an option' },
                        { value: 'talent', label: 'Talent' },
                        { value: 'recruiter', label: 'Recruiter' },
]
```
so options = {userTypeData}

**Review by mosidrum — CHANGES_REQUESTED** [review]

**Review by mosidrum — APPROVED** [review]
# Good Job Boss 

approved-target-lady
